src/Tools/Source/RunTests/Options.cs

```diff
@@ -161,7 +168,7 @@ internal class Options
             {
                 { "dotnet=", "Path to dotnet", (string s) => dotnetFilePath = s },
                 { "configuration=", "Configuration to test: Debug or Release", (string s) => configuration = s },
-                { "tfm=", "Target framework to test", (string s) => targetFrameworks.Add(s) },
+                { "tfm=", "Target framework to test: both, core, framework", (TestTargetFramework t) => testTargetFramework = t},
```

Comment:
I'm not a _huge_ fan of naming this tfm, since it immediately reads to me that valid TFM strings shoudl be valid arguments, but I'm not going to block on it.

Reply:
I was thinking `runtime`

Reply:
Or `testRuntime`

Reply:
I don't have any objections to those names. This PR though is blocking several other infra PRs. Plan is to merge this as is assuming it's green. If it's red I'll incorporate the name suggestion on the next push, if it's green I'll try and incorporate it in one of the other infra PRS stacked behind it.

Reply:
Understood. That's why I didn't block.